src/main/java/bio/terra/service/notification/PubSubService.java

```diff
@@ -0,0 +1,33 @@
+package bio.terra.service.notification;
+
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.cloud.pubsub.v1.TopicAdminClient;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.Topic;
+import com.google.pubsub.v1.TopicName;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PubSubService {
+  private static final Logger logger = LoggerFactory.getLogger(PubSubService.class);
+
+  public void createTopic(String projectId, String topicId) throws IOException {
+    try (TopicAdminClient topicAdminClient = TopicAdminClient.create()) {
+      TopicName topicName = TopicName.of(projectId, topicId);
```

Comment:
what does this represent? What do projectId and topicId represent?

Reply:
Rae, I was confused about this too - I found this documentation helpful as well in case you wanted to see it visually: https://cloud.google.com/pubsub/docs/pubsub-basics